modules/openxr/SCsub

```diff
@@ -82,6 +104,7 @@ env_openxr.add_source_files(module_obj, "action_map/*.cpp")
 # We're a little more targeted with our extensions
 if env["platform"] == "android":
     env_openxr.add_source_files(module_obj, "extensions/openxr_android_extension.cpp")
+    env_openxr.AppendUnique(CPPDEFINES=["XR_OS_ANDROID", "XR_USE_PLATFORM_ANDROID"])
```

Comment:
I was looking that this before and I think a nicer solution is to move the original check that adds the defines  to `env_openxr` before `env_thirdparty` is cloned from it so we have consistent settings here.

Reply:
Moved as requested.